MediaBrowser.Providers/Lyric/LrcLyricParser.cs

```diff
@@ -3,34 +3,29 @@
 using System.Globalization;
 using System.IO;
 using System.Linq;
-using System.Threading.Tasks;
+using Jellyfin.Extensions;
 using LrcParser.Model;
 using LrcParser.Parser;
-using MediaBrowser.Controller.Entities;
 using MediaBrowser.Controller.Lyrics;
 using MediaBrowser.Controller.Resolvers;
-using Microsoft.Extensions.Logging;
 
 namespace MediaBrowser.Providers.Lyric;
 
 /// <summary>
-/// LRC Lyric Provider.
+/// LRC Lyric Parser.
 /// </summary>
-public class LrcLyricProvider : ILyricProvider
+public class LrcLyricParser : ILyricParser
 {
-    private readonly ILogger<LrcLyricProvider> _logger;
-
     private readonly LyricParser _lrcLyricParser;
 
+    private static readonly string[] _supportedMediaTypes = { "lrc", "elrc" };
```

Comment:
Wouldn't it be simpler if you included a dot here? That way you won't have to remove it from the GetExtension calls